tools/tiff2pcd.cpp

```diff
@@ -394,10 +392,6 @@ int main(int argc, char ** argv)
           // Continue with the next depth entry
           continue;
         }
-        if(found == 0)
```

Comment:
I think this is rather a bug. This check `found == 0` and following console output should happen outside of the loop.

Reply:
Adjusted